prism-agent/service/server/src/main/scala/io/iohk/atala/agent/server/config/AppConfig.scala

```diff
@@ -166,9 +166,20 @@ final case class AgentConfig(
 
 }
 
-final case class HttpEndpointConfig(http: HttpConfig, publicEndpointUrl: String)
+final case class HttpEndpointConfig(http: HttpConfig, publicEndpointUrl: String) {
+  require(isValidUrl(publicEndpointUrl), s"Invalid URL: $publicEndpointUrl")
```

Comment:
Other config objects has this `def validate: Either` method, should we keep this consistent with other validations?

Reply:
+1 @patlo-iog let's make it consistent with what we have in [AppConfig](https://github.com/hyperledger-labs/open-enterprise-agent/blob/320ab6a876606eb68f48fe7d78983b4e044b5084/prism-agent/service/server/src/main/scala/io/iohk/atala/agent/server/config/AppConfig.scala#L36) for instance.